include/UploadFile.php

```diff
@@ -51,32 +51,63 @@
  */
 class UploadFile
 {
-    public $field_name;
-    public $stored_file_name;
-    public $uploaded_file_name;
-    public $original_file_name;
-    public $temp_file_location;
-    public $use_soap = false;
-    public $file;
-    public $file_ext;
-    public $mime_type;
+    /**
+     * @var string
+     */
     protected static $url = "upload/";
-
     /**
      * Upload errors
      * @var array
      */
     protected static $filesError = array(
         UPLOAD_ERR_OK => 'UPLOAD_ERR_OK - There is no error, the file uploaded with success.',
-        UPLOAD_ERR_INI_SIZE => 'UPLOAD_ERR_INI_SIZE - The uploaded file exceeds the upload_max_filesize directive in php.ini.',
-        UPLOAD_ERR_FORM_SIZE => 'UPLOAD_ERR_FORM_SIZE - The uploaded file exceeds the MAX_FILE_SIZE directive that was specified in the HTML form.',
+        UPLOAD_ERR_INI_SIZE => 'UPLOAD_ERR_INI_SIZE - The uploaded file exceeds the upload_max_filesize
```

Comment:
Why breaking language lines?

Reply:
Lines should be no longer than 120 characters.

On the other hand, this doesn't affect the result, the line break will not be included in the resulting string.

And yes those seem to be hard coded. But I don't know if they are actually being used anywhere or if that is just dead code...

Reply:
Searching for it here: https://github.com/salesagility/SuiteCRM/search?utf8=%E2%9C%93&q=UPLOAD_ERR_INI_SIZE&type=
Note: line breaks "WHEN" in language files do matter as the translation platform makes that a real line break and makes the task harder for translators. Not the case here as this is not a language file.

Reply:
Thanks for the heads up regarding line breaks in language files.